iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java

```diff
@@ -91,10 +112,8 @@ void shouldNotParseYamlFileWithIncompleteIdentifiers() {
     assertNotSourceFileIsParsed();
 
     var logs = logTester.logs(Level.DEBUG);
-    assertThat(logs).hasSize(2);
+    assertThat(logs).hasSize(1);
     assertThat(logs.get(0))
-      .isEqualTo("Skipping initialization of Helm processor");
```

Comment:
Shouldn't here be the info log `"Skipping initialization of Helm processor"`? because internal.helm.enable is set to false?